dts/arm/st/f2/stm32f2.dtsi

```diff
@@ -216,7 +216,7 @@
 			num-bidir-endpoints = <4>;
 			ram-size = <1280>;
 			maximum-speed = "full-speed";
-			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x20000000>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x80>;
```

Comment:
Please keep the 8 digits format (0x00000080).

Reply:
Done